NewSpring/ApollosProject
# (DEV-4150) Fixes Sharing URLs

## DESCRIPTION

<img width="1428" alt="Screen Shot 2019-10-14 at 1 29 03 PM" src="https://user-images.githubusercontent.com/2659478/66771191-8754fb80-ee87-11e9-9f29-ef49e6bc2502.png">


### What does this PR do, or why is it needed?

Adds parents to sermon and devotional sharing URLs so they work on the website.

### How do I test this PR?

1. Run this:

```
{
  userFeed {
    edges {
      node {
        sharing {
          url
        }
      }
    }
  }
}
```

2. Share a piece of content and see if it opens:

- on the app
- on the browser

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
